changelog/new_introduce_enforced_style_for_multiline_diff_comma_20250208163732.md

```diff
@@ -0,0 +1 @@
+* [#9935](https://github.com/rubocop/rubocop/issues/9935): Introduce EnforcedStyleForMultiline "diff_comma". ([@flavorjones][https://github.com/flavorjones])
```

Comment:
This should make CI pass:
```suggestion
* [#9935](https://github.com/rubocop/rubocop/issues/9935): Introduce EnforcedStyleForMultiline "diff_comma". ([@flavorjones][])
```

Reply:
Ugh! Thanks for pointing that out. Will fix.